### ISSUE TITLE
Turn off modifier

### ISSUE THREAD

**Issue by blond** [bem-sdk/bem-entity-name#42]
Now `bem-naming` turn off modifier if modifier value is falsy.

**Actual `BemEntityName`:**

``` js
const entityName = new BemEntityName({ block: 'block', mod: { name: 'mod', val: false } });

entity.mod // { name: 'mod', val: null }
entity.type // mod
```

**`bem-naming` expected:**

``` js
const entityName = new BemEntityName({ block: 'block', mod: { name: 'mod', val: false } });

entity.mod // {}
entity.type // block
```

**Comment by blond** [comment]
@tadatuta @zxqfox what do you think?

Is it necessary to repeat the same behavior?

**Comment by zxqfox** [comment]
I'm not sure what these two record means:
```js
{ block: 'block', mod: { name: 'mod', val: false } }
{ block: 'block', mod: {} })
```
At the moment we need 3 states: some specific modifier, any modifier, and no modifier:
```js
{ mod: 'm', val: 'v' }
{ mod: 'm', val: true } or { mod: 'm' }
{ }
```

`val: false` is more than we need at my pov.

**Comment by blond** [comment]
> { block: 'block', mod: {} })

It is mean `{ block: 'block' }`.

> { block: 'block', mod: { name: 'mod', val: false } }

In `bem-naming` it is mean `{ block: 'block' }`.

Now in `bem-entity-name` it is mean `{ block: 'block', mod: { name: 'mod', val: false } }`.

**Comment by zxqfox** [comment]
> Now in bem-entity-name it is mean { block: 'block', mod: { name: 'mod', val: false } }.
What the heck is that?

**Comment by blond** [comment]
> What the heck is that?

Modifier with `false` value.

**Comment by blond** [comment]
We decided not to change the normalize logic. If necessary, this can be done in higher-level packages.

**Comment by zxqfox** [comment]
Actual:
```js
> q = new qq({ block: 'block', mod: { name: 'mod', val: false } });
BemEntityName { block: 'block', mod: { name: 'mod', val: undefined } }
> q.mo
q.mod      q.modName  q.modVal

> q.mod
{ name: 'mod', val: undefined }
> q.id
'block_mod'
>
```

**Comment by zxqfox** [comment]
We decided to assert modifier value: it should be nothing, or a string, or true.

**Comment by blond** [comment]
Closed in favor of #80
